__init__.py

```diff
@@ -483,8 +483,7 @@ def _handle_sensor(self, message):
                 sensor_name, sensor_state, sensor_unit)))
             if len(quantity) > 0:
                 quantity = quantity[0]
-                if (quantity.unit.name != "dimensionless" and
-                        quantity.uncertainty <= 0.5):
+                if (quantity.unit.name != "dimensionless"):
```

Comment:
Why do we no longer care about the `quantity.uncertainty`? 

The comments a few lines up suggest this code block is optional, so I'm guessing the intention was only to use it if quantulum had a reasonable confidence in it's parsing attempt?

**Sidenote and not required** - It might be useful to refactor this quantulum block out to it's own method to make it easier to understand what it's doing and why. Even though it's not yet re-used elsewhere it feels like a distinct function.

Reply:
oh, I just read the comment further up in the conversation that this is not about confidence at all. So ignore that sorry.

Reply:
Can we resolve this discussion?

Reply:
I think Yes :)